### PR TITLE
Hotfixes for TestProxy LiveTesting

### PR DESCRIPTION
**Pull request by scbedd**
@rakshith91

### CODE REVIEW

**Review by rakshith91 — APPROVED** [review]
LGTM

File: eng/pipelines/templates/stages/archetype-sdk-tests.yml [hunk 1]
```diff
@@ -86,6 +86,9 @@ parameters:
   - name: PlatformPostSteps
     type: object
     default: []
+- name: TestProxy
```
Comment (sima-zhu) [inline eng/pipelines/templates/stages/archetype-sdk-tests.yml:89]: The alignment here is not correct, which causes the live test pipeline failed.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1204102&view=logs&j=96791242-dbf3-587e-3a06-ae5af5c1a705&t=68bc2dc9-cd55-531a-7fab-f5d2306827b9

**Comment by v-xuto** [comment]
@rakshith91 There is a problem with indentation in file `eng/pipelines/templates/stages/archetype-sdk-tests.yml`, there should be a space in front. Maybe it should be changed as shown below. 
The same problem also exists in file `eng/pipelines/templates/jobs/live.tests.yml`.  Please take a look.
image

I found this error when I ran the pipeline.
image

**Comment by scbedd** [comment]
@v-xuto this was not merged by me, apologies for the issues. Thanks @rakshith91 for reverting the original merge.

**Pull request merged**